pulp_ansible/app/tasks/signature.py

```diff
@@ -91,13 +175,36 @@ def sign(repository_href, content_hrefs, signing_service_href):
     SigningDeclarativeVersion(first_stage, repository).create()
 
 
+def sigstore_sign(repository_href, content_hrefs, sigstore_signing_service_href):
```

Comment:
Also here, why adding another singing task. The existing one should just create signatures based on the type of configured SigningService. In an ideal world there would not even be any distinction here. `singing_service.sign()` would just return the desired type of signature.
Or maybe a `sign_collection(cv, signing_service)` function for that matter.